modules/Forum/classes/Forum.php

```diff
@@ -668,4 +668,23 @@ public function getAnySubforums(int $forum_id, array $groups = [0], int $depth =
 
         return $ret;
     }
+
+    /**
+     * @param array $labels  of label ids
+     * @param array $user_groups array of a user their group ids
+     * @return array An array of the ids of the labels the user has access to
+     */
+    static function getAccessibleLabels(array $labels, array $user_groups): array {
```

Comment:
Can we add the `public` access modifier here please - always good to be specific